core/src/impl/Kokkos_Utilities.hpp

```diff
@@ -446,6 +447,26 @@ struct is_specialization_of<Template<Args...>, Template> : std::true_type {};
 // </editor-fold> end is_specialization_of }}}1
 //==============================================================================
 
+//==============================================================================
+// <editor-fold desc="Folding emulation"> {{{1
+
+// acts like void for comma fold emulation
+struct _fold_comma_emulation_return {};
+
+template <class... Ts>
+constexpr KOKKOS_INLINE_FUNCTION _fold_comma_emulation_return
+emulate_fold_comma_operator(Ts&&...) noexcept {
+  return _fold_comma_emulation_return{};
+}
+
+#define KOKKOS_IMPL_FOLD_COMMA_OPERATOR(expr)                                \
```

Comment:
Can you give an example here for the purpose of this macro?

Reply:
https://github.com/dhollman/kokkos/blob/bf4e2f32316984eb582e32a70e46556b424b0e25/core/src/Kokkos_Graph.hpp#L219-L220